app/routes/app/components/App.js

```diff
@@ -110,13 +97,10 @@ class App extends Component {
 }
 
 App.propTypes = {
-  modal: PropTypes.object.isRequired,
-  ticker: PropTypes.object.isRequired,
   form: PropTypes.object.isRequired,
   formProps: PropTypes.object.isRequired,
   closeForm: PropTypes.func.isRequired,
   error: PropTypes.object.isRequired,
-  info: PropTypes.object.isRequired,
```

Comment:
One question is should I include these even if they are not used in the component itself, given the children sometimes or always do? I presume not.

Reply:
yeah no need. we used to manually pass objects like `info` into child components within `App.js` _but_ I've since started to refactor with a much cleaner approach by creating prop objects in Containers. Example: https://github.com/LN-Zap/zap-desktop/blob/master/app/routes/app/containers/AppContainer.js#L193